src/csharp/Microsoft.Spark/Versions.cs

```diff
@@ -12,6 +12,8 @@ internal static class Versions
         internal const string V2_3_3 = "2.3.3";
         internal const string V2_4_0 = "2.4.0";
         internal const string V2_4_2 = "2.4.2";
+        internal const string V2_4_4 = "2.4.4";
+        internal const string V2_4_5 = "2.4.5";
```

Comment:
Why are these added?

Reply:
If i remerber right,  E2E test was failed on stage V2_4_4 and V_2_4_5 

I've pushed new commit which removed V_2_4._4 V_2_4_5 

let's see CI result

Reply:
I got it wrong,  it looks good.